dsa-solutions/lc-solutions/0500-0599/0583-Delete-operations-for-2-strings.md

```diff
@@ -0,0 +1,159 @@
+---
+id: delete operations for 2 strings
```

Comment:
replace `id: delete operations for 2 strings` to `id: delete-operations-for-2-strings`

Reply:
@Nikhil11200 update your code

Reply:
@Ajay-Dhangar updated the id name , plz check and merge my PR.